src/main/java/tc/oc/pgm/api/setting/SettingValue.java

```diff
@@ -21,7 +21,9 @@
   DEATH_ALL("death", "all"), // Send all death messages, highlight your own
   PICKER_AUTO("picker", "auto"), // Display after cycle, or with permissions.
   PICKER_ON("picker", "on"), // Display the picker GUI always
-  PICKER_OFF("picker", "off"); // Never display the picker GUI
+  PICKER_OFF("picker", "off"), // Never display the picker GUI
+  OBSERVERS_SHOWN("observers", "shown"), // Show observers
+  OBSERVERS_HIDDEN("observers", "hidden"); // Hide observers
```

Comment:
Why not just on/off? I think it's better to be somewhat consistent with other settings if players want to be able to find/explore them easily.

Reply:
It's currently more understandable what it means.

Reply:
Are you sure? I think obs on or obs off is just as easy to understeand as obs shown and obs hidden. It's just a matter of consistency with other settings tho, i am not too concerned about that as long as it's easy to tab-complete tho

Reply:
Let's stick to on/off. Eventually we'll have a Gui with an explanation of what the setting does